bundles/org.openhab.binding.darksky/README.md

```diff
@@ -61,9 +63,9 @@ Once the parameters `forecastHours` or `forecastDays` will be changed, the avail
 | current          | gust-speed         | Number:Speed         | Current gust speed. **Advanced**                                        |
 | current          | cloudiness         | Number:Dimensionless | Current cloudiness.                                                     |
 | current          | visibility         | Number:Length        | Current visibility.                                                     |
-| current          | rain               | Number:Length        | Rain volume of the last hour.                                           |
-| current          | snow               | Number:Length        | Snow volume of the last hour.                                           |
-| current          | precip-intensity   | Number:Length        | Current precipitation intensity.                                        |
+| current          | rain               | Number:Speed         | Rain volume of the last hour.                                           |
```

Comment:
Precipitation is measured in l/m2 (or more commonly mm). This is volume/area, not speed (distance/time). The [german Wikipedia article](https://de.wikipedia.org/wiki/Niederschlag#Niederschlagsmenge) also supports that . If you bring the time in, it's a flow (volume/time).

Why did you change it?

Reply:
The API returns "`precipIntensity`: Millimeters per hour." (see https://darksky.net/dev/docs#forecast-request). That fits per definition fit the dimension speed. I know it sounds weird in the first moment. To make it more clear I should maybe adopt the descriptions and translations too.

Reply:
Being a physicist by trade I'm nitpicking here xD. In fact it's volume/(area*time), which is NOT speed, even if the dimension fits (the old `Length` was wrong in the same way). But ok.

Reply:
Done.